### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by DrPandemic**
- Add instructions in README.md
- Remove some unused code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,25 @@ Bitcoin Mining Pool
...
 ### Install
 `make install` (possibly requires `sudo`)
 ### Run tests
 - Unit tests: `make test`
 - Unit tests with branch coverage: `make coverage`
+
+## Server
+### Install
+- Install the system library `zeromq`
+- Install an up to date node
```
Comment (JesseEmond) [inline README.md]: I think it's [up-to-date](https://en.wiktionary.org/wiki/up-to-date) (dashes).

File: README.md [hunk 2]
```diff
@@ -5,9 +5,25 @@ Bitcoin Mining Pool
...
+- `npm install -g typescript`
+- `npm install -g tsd`
+- `npm install -g gulp`
+- Inside `server/` run `tsd install` and `npm install`
+### Run
+- Inside `server/`, build with the command `gulp`
+- Inside `server/`, run with the commande `node release/src/app.js`
```
Comment (JesseEmond) [inline README.md:29]: Isn't it just `node release/app.js`? That's what I'm doing here.

**Comment by JesseEmond** [comment]
Some comments, but the rest LGTM :+1: Thanks for taking the time.

I think we should make a PR to show how we'd test the whole system (running `generate-bitcoins`,  the miner, etc.) I'll do that later.

File: README.md (hunk 2 above)
Comment (DrPandemic) [inline README.md:29]: Really? I tried to do this, but I was unable to achieve that. Mine is definitely `release/src/app.js`

Comment (JesseEmond) [inline README.md:29]: @DrPandemic I am confused. All my `.js` files are directly in `release`, no `src` subfolder. Not sure why we'd be getting a different folder hierarchy.. I guess you can merge anyway then.

**Pull request merged**